test/Ocelot.AcceptanceTests/Steps.cs

```diff
@@ -214,30 +214,12 @@ public void ThenTheResponseBodyHeaderIs(string key, string value)
 
         public void GivenOcelotIsRunningReloadingConfig(bool shouldReload)
         {
-            _webHostBuilder = new WebHostBuilder();
-
-            _webHostBuilder
-                .ConfigureAppConfiguration((hostingContext, config) =>
-                {
-                    config.SetBasePath(hostingContext.HostingEnvironment.ContentRootPath);
-                    var env = hostingContext.HostingEnvironment;
-                    config.AddJsonFile("appsettings.json", optional: true, reloadOnChange: false)
-                        .AddJsonFile($"appsettings.{env.EnvironmentName}.json", optional: true, reloadOnChange: false);
-                    config.AddJsonFile("ocelot.json", optional: false, reloadOnChange: shouldReload);
-                    config.AddEnvironmentVariables();
-                })
-                .ConfigureServices(s =>
-                {
-                    s.AddOcelot();
-                })
-                .Configure(app =>
-                {
-                    app.UseOcelot().Wait();
-                });
-
-            _ocelotServer = new TestServer(_webHostBuilder);
+            StartMinimalTestServerWithOcelotConfiguration((hostingContext, config) => config.AddJsonFile("ocelot.json", optional: false, reloadOnChange: shouldReload));
+        }
 
-            _ocelotClient = _ocelotServer.CreateClient();
+        public void WhenOcelotIsRunningMergedConfig(MergeOcelotJson mergeOcelotJson)
+        {
+            StartMinimalTestServerWithOcelotConfiguration((hostingContext, config) => config.AddOcelot(hostingContext.HostingEnvironment, mergeOcelotJson));
```

Comment:
How many references does this method have?

If it has one reference only, consider moving the helper to the `ConfigurationMergeTests` class as public static definition plz.

Reply:
not an issue!